docs/reference/contributing/target/rtos.md

```diff
@@ -1,6 +1,6 @@
 ### RTOS
 
-CMSIS/RTX code is imported from <a href="https://github.com/ARM-software/CMSIS_5/" target="_blank">the original CMSIS repository</a>.
+CMSIS/RTX code is imported from <a href="https://github.com/ARM-software/CMSIS_5/" target="_blank">the original CMSIS repository</a> with the help of Python script in the `mbed-os` repository `mbed-os\tools\importer\importer.py`.
```

Comment:
original should be 'upstream development'

Reply:
further down there is generic system would should be 'platform operating system'